tests/caffe2_ref_test.py

```diff
@@ -46,8 +44,8 @@ def test_relu_node_inplace(self):
         graph_def = make_graph(
             [node_def],
             name="test",
-            inputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT, [3, 2])],
-            outputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT, [3, 2])])
+            inputs=[make_tensor_value_info("X", onnx.onnx_pb2.TensorProto.FLOAT, [3, 2])],
```

Comment:
No, please don't do this. `onnx_pb2` is supposed to be an internal implementation detail (and in fact, I have a PR which renames what this is called.) If lint is complaining about it, we need to add something like `__all__` to `onnx` so that the linter knows what is being brought into scope.